finance/pta/Portfile

```
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               perl5 1.0
+
+perl5.require_variant   yes
+
+name                    pta
+version                 0.1
+categories              finance
+platforms               {darwin any}
+supported_archs         noarch
+maintainers             @admacleod
+description             the plain text accounting program
+long_description        pta reads bookkeeping journals and writes account lists, \
+                        balances, and cost center assessments to standard output.
+license                 ISC
+homepage                https://mandoc.bsd.lv/pta/
+master_sites            https://mandoc.bsd.lv/pta/release/
```

Tiny nitpick: this could also be specified to use the `homepage` variable:

```suggestion
master_sites            ${homepage}release/
```